.travis.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         - mkdir -p s2n/test-deps/saw/bin
         - cp bin/saw s2n/test-deps/saw/bin
         - cd s2n
-        - git checkout fc861ca412cd50f3b1d264f940e6251aef63c94d
+        - git checkout 28ea0fea9ab7dc710c9424d9b869b39ca3804d79
```

Comment:
this commit is not on master because it disables some verification tasks that are exceed the time/memory limits of Travis. I rebased it on top of GaloisInc/s2n@16b61f091e7da90859a38a366d16652b7c8aab4c, now it should work

Reply:
Interesting, that seems like it got us somewhat unstuck, now we have a new failure mode in BIKE:

```
[03:38:42.067] Verifying is_new ...
4168[03:38:42.073] Simulating is_new ...
4169[03:38:42.690] Symbolic simulation completed with side conditions.
4170[03:38:43.146] Checking proof obligations is_new ...
4171Calling Yices to check sat
4172Running check sat
4173Calling Yices to check sat
4174Running check sat
4175[03:38:43.202] You have encountered a bug in SawCore's implementation.
4176*** Please create an issue at https://github.com/GaloisInc/saw-core/issues
4177
4178%< --------------------------------------------------- 
4179  Revision:  10b799c885e5b5d90ce05f9456ccd6262d0a8190
4180  Branch:    HEAD
4181  Location:  Verifier.SAW.Simulator.Value.apply
4182  Message:   Not a function value:
4183             <<array>>
4184CallStack (from HasCallStack):
4185  panic, called at src/Verifier/SAW/Utils.hs:37:9 in saw-core-0.1-inplace:Verifier.SAW.Utils
4186  panic, called at src/Verifier/SAW/Simulator/Value.hs:219:14 in saw-core-0.1-inplace:Verifier.SAW.Simulator.Value
4187%< --------------------------------------------------- 
4188
4189Makefile:129: recipe for target 'bike_r2/verify_bike_r2.log' failed
```

Reply:
yes, see the comment about the Crucible submodule

Reply:
Oh, I see what you mean now

Reply:
Thanks for your help getting this sorted out Andrei!